docs/pages/guides/using-eslint.mdx

```diff
@@ -167,6 +167,29 @@ To install Prettier in your project:
 
 ### Setup
 
+<Tabs>
+
+<Tab label="Flat Config">
```

Comment:
```suggestion
<Tab label="Flat config">
```

I am not sure this needs to be in title case. Checked ESLint documentation for reference.